hack/lima-and-qemu.pl

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+
+my $install_dir = "/usr/local";
+
+# This script creates a tarball containing lima and qemu, plus all their
+# dependencies from /usr/local/**.
+#
+# Dependencies are collected via DTrace while running:
+# `limactl start examples.alpine.yaml; limactl stop alpine; limactrl delete alpine`.
+#
+# New processes (with their command line arguments) have been captured by
+# `sudo dtrace -s /usr/bin/newproc.d` (on a system with SIP disabled, using lima 0.3.0):
+#
+# 5680 <777>  limactl start --tty=false examples/alpine.yaml
+# 5681 <5680> curl -fSL -o /Users/jan/Library/Caches/lima/download/by-url-sha256/21753<...>
+# 5683 <5680> qemu-img create -f qcow2 /Users/jan/.lima/alpine/diffdisk 107374182400
+# 5684 <5680> /usr/local/bin/limactl hostagent --pidfile /Users/jan/.lima/alpine/ha.pid alpine
+# 5686 <5684> ssh-keygen -R [127.0.0.1]:60020 -R [localhost]:60020
+# 5687 <5684> ssh -o ControlMaster=auto -o ControlPath=/Users/jan/.lima/alpine/ssh.sock -o <...>
+# 5685 <5684> /usr/local/bin/qemu-system-x86_64 -cpu Haswell-v4 -machine q35,accel=hvf -smp <...>
+# 5689 <5684> ssh -o ControlMaster=auto -o ControlPath=/Users/jan/.lima/alpine/ssh.sock -o <...>
+# ... many more ssh sub-processes like the one above ...
+# 5800 <777>  limactl stop alpine
+# 5801 <5684> ssh -o ControlMaster=auto -o ControlPath=/Users/jan/.lima/alpine/ssh.sock -o <...>
+# 5896 <777>  limactl delete alpine
+#
+# It shows the following binaries from /usr/local are called:
+
+record("$install_dir/bin/limactl");
+record("$install_dir/bin/qemu-img");
+record("$install_dir/bin/qemu-system-x86_64");
```

Comment:
Also aarch64 (in future)

Reply:
> Also aarch64 (in future)

Yes, but still waiting to get an M1 mini for testing before I make those changes. Also homebrew installs to `/opt/homebrew` on Apple Silicon, so I wonder what other things may need to be adjusted.

Reply:
Or did you mean you want to bundle `qemu-system-aarch64` for the Intel tarball? I don't think that has practical value; it just takes too long to emulate the whole OS.

I'm planning to do `aarch64` support on `x86_64` (and vice versa) via `qemu-user-*` and `binfmt_misc`, so only the container code needs to be emulated, not the whole OS.